addons/sale_project/models/sale_order.py

```diff
@@ -155,6 +157,29 @@ def _auto_init(self):
             """)
         return super()._auto_init()
 
+    @api.depends('product_id')
+    def _compute_qty_delivered_method(self):
```

Comment:
See with @IT-Ideas but I think we need to do this compute in a migration script and also the quantity delivered field.  🙂